### DISCUSSION TITLE
Passing a prop from the content.md to the parent Vue page?

### DISCUSSION THREAD

**Discussion by shintaii** [nuxt/content#1971] [Q&A]
Maybe i'm doing it wrong, but currentle i have a [...slug].vue page for all content pages. 

That vue page is a template of some divs, some styling and a contentdoc component. But i also want to have an image besides the content. Easy peasy to do. 

But what i really want is that the content writer, working in the *.md files, to pass a prop to the [...slug].vue. With an image url. I can pass props to the components i call in the .md file (using the MDC syntax). 

In this case i want to pass the imageUrl to the parent ([...slug].vue) so that i can render the image next to the content, or when imageUrl is undefined, it is a 1 column layout. 

No idea how to do it, but maybe i'm missing something :)

**Comment by nobkd** [comment]
In doc driven mode, you can use the `page` key from the `useContent()` composable in the `slug` page to get a key from the Markdown front-matter.
The value of this key can be the image link, which you then use in the content of the `slug` page.

[Example on Stackblitz](https://stackblitz.com/edit/github-3n6syx?file=pages%2F[...slug].vue,content%2Findex.md,content%2Fabout.md)
